packages/core/utils/lib/sanitize/sanitizers.js

```diff
@@ -16,7 +15,13 @@ const sanitizePrivates = curry((schema, entity) => {
 });
 
 const defaultSanitizeOutput = curry((schema, entity) => {
-  return pipeAsync(sanitizePrivates(schema), sanitizePasswords(schema))(entity);
+  const defaultSanitize = async (entity) => {
+    removePassword(schema, entity);
+    removePrivate(schema, entity);
+    return entity;
```

Comment:
So it doesn't call `traverseEntity` anymore ?

Reply:
I think you are looking at an old commit, it's calling the traverseEntity below that piece of code.

Reply:
Indeed it was on a another commit! Nice, so it traverse only once instead of twice. Makes sense to me!

Reply:
exactly, and pbbly we could try to unify the rest of traversals if there are